developer/src/kmc-copy/src/KeymanProjectCopier.ts

```diff
@@ -131,6 +130,20 @@ export class KeymanProjectCopier implements KeymanCompiler {
       return null;
     }
 
+    if(project.isKeyboardProject()) {
+      // LDML and .kmn keyboards have the same id validity requirements, so
+      // we'll just test against .kmn
```

Comment:
Even if they have the same id validity requirements, should we test separately so it can report `ERROR_InvalidLdmlKeyboardId`?

Is there a branch for `else if(project.isLdmlKeyboardProject())` ?

Reply:
The tricky thing is that the project is either a keyboard project, or a lexical model project. We don't differentiate at the project level between kmn or ldml. So that's why I opted to go this way.